components/06-examples/header.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="dso-logo-bar">
     {{> '@logo' }}
     <div class="dso-tools-bar">
-      {{render '@searchbar' searchbarContext merge=true }}
+      
```

Comment:
Deze witregel voegt niets toe

Reply:
Deze is uitgevoerd. Zo niet corrigeer me waar nodig.